views/register.hbs

```diff
@@ -0,0 +1,52 @@
+<div class="login">
+    <section>
+        <div class="container">
+            <div class="login__form">
+                <div class="flex_box">
+                    <div class="form_text">
+                        <h2 tabindex="0">Login</h2>
+                        <p tabindex="0">Get access to your Orders , Wishlists and
+                            Recommendations </p>
+                    </div>
+                    <div class="form_set">
+                        <form autocomplete="off">
+                            <div class="fieldset">
+                                <input type="text" class="form-control"
```

Comment:
name is mandatory attribute for a form field. Please find out why and explain here.

Reply:
the name attribute defines the name of the control, that is why we use name attribute for an input tag.